DanyloVoloshyn/seventh_homework/third_task.py

```diff
@@ -0,0 +1,20 @@
+def count_characters(s):
```

Comment:
good job
look at this version of code:
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result